include/ibamr/KrylovLinearSolverStaggeredStokesSolverInterface.h

```diff
@@ -89,6 +89,12 @@ class KrylovLinearSolverStaggeredStokesSolverInterface : public StaggeredStokesS
      */
     void setVelocityPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& U_problem_coefs);
 
+	/*!
+	 * \brief Set if velocity and pressure have nullspace.
+	 */
+	void setComponentsHaveNullspace(const bool has_velocity_nullspace,
```

Comment:
(1) I think so, because the implementations are capable of generating vectors themselves.
(2)  setComponentNullspaces() sounds like we are passing vectors, but I just to pass booleans..